modules/rest-api/pages/rest-get-cluster-events.adoc

```diff
@@ -1 +1,143 @@
 = Getting Cluster Events
+
+:description: pass:q[System events that occur on the cluster can be retrieved.]
+:page-topic-type: reference
+
+[abstract]
+{description}
+
+== HTTP method and URIs
+
+----
+GET /events
+
+GET /eventsStreaming
+----
+
+[#rest-getting-system-events-description]
+== Description
+
+The `events` URI returns previously logged system events.
+The `eventsStreaming` URI returns previously logged system events as an array; and then streams all further system events as they occur.
+
+== Curl Syntax
+
+----
+curl -v -X GET -u [username]:[password]
+  http://<ip-address-or-domain-name>:8091/events?sinceTime=<time-value>&limit=<limit-value>
+
+curl -v -X GET -u [username]:[password]
+  http://<ip-address-or-domain-name>:8091/eventsStreaming
+----
+
+The `sinceTime` path-parameter specifies the time (inclusive) from which previously logged events are to be returned.
+The associated `time-value` must be specified in _ISO 8601 UTC_ format &#8212; which is `YYYY-MM-DDThh:mm:ssZ`.
+If this path-parameter is not used, logs starting from the oldest are returned, .
+
+The `limit` path-parameter specifies the maximum number of logged events to be returned.
+The default `limit-value` is `250`.
+If `limit-value` is specified as `1`, no limit is imposed; and all events logged since the specified `sinceTime`, or from the oldest logged event, are therefore returned.
```

Comment:
Typo: "is specified as '1'" should be "is specified as '-1'".

Reply:
I think we should included another example for the following case:
if 'sinceTime' is not passed in the query params then the most recent 'limit-value' number of event logs are returned.
/events?limit=300 will send the most recent 300 event log.

Reply:
Okay, I've tried to clarify this a bit. I've added the extra example, and have also added some words in the main description of the params. Let me know if this seems right. Thanks.

Reply:
The changes look good @tonyjhillman. Thanks.

Reply:
Thanks. I'll merge.